src/content/en/2021/security.md

```diff
@@ -1099,54 +1109,60 @@ We now turn to cryptominer market share in the following figure.
+<a hreflang="en" href="https://datatracker.ietf.org/doc/html/draft-foudil-securitytxt-12">`security.txt`</a> is a file format for websites to provide a standard for vulnerability reporting. Website providers can provide contact details, PGP key, policy, and other information in this file. White hat hackers can then use this information to conduct security analyses on these websites or report a vulnerability.
 
 {{ figure_markup(
   image="security-usage-of-well-known-security.png",
   caption="Use of `security.txt`.",
-  description="Bar chart showing 4.93% of websites in desktop and 4.89% of websites in mobile have `security.txt` endpoint.",
+  description="Bar chart showing 4.9% of websites in desktop and 4.9% of websites in mobile have `security.txt` endpoint.",
   chart_url="https://docs.google.com/spreadsheets/d/e/2PACX-1vR38ZfiZXxdGuzG4ywCEPKIU1Wl6E2bZwcQPyALavGq5q948gmWb8sT-Xo5T6K5z8smKPg6EKxV0JUI/pubchart?oid=1958147881&format=interactive",
   sheets_gid="399976976",
   sql_file="well-known_security.sql"
   )
 }}
 
-And the following figure shows that the policy is the most used property of utilized `security.txt` files.
+We see that just under 5% of the websites already use this standard—although it is still a very young standard.
+
+And the following figure shows that the policy is the most used property of utilized `security.txt` files:
 
 {{ figure_markup(
   image="security-usage-of-properties-in-well-known-security.png",
   caption="Use of security.txt properties.",
-  description="Bar chart showing use of different properties in security.txt. `signed` is 0.37% in desktop and 0.28% in mobile, Canonical is 2.75% in desktop and 2.72% in mobile, Encryption is 2.55% in desktop and 2.13% in mobile, Expires is 0.93% in desktop and 0.70% in mobile, Policy is 6.54$ in desktop and 6.39% in mobile.",
+  description="Bar chart showing use of different properties in security.txt. `signed` is 0.4% in desktop and 0.3% in mobile, Canonical is 2.8% in desktop and 2.7% in mobile, Encryption is 2.6% in desktop and 2.1% in mobile, Expires is 0.9% in desktop and 0.7% in mobile, Policy is 6.5% in desktop and 6.4% in mobile.",
   chart_url="https://docs.google.com/spreadsheets/d/e/2PACX-1vR38ZfiZXxdGuzG4ywCEPKIU1Wl6E2bZwcQPyALavGq5q948gmWb8sT-Xo5T6K5z8smKPg6EKxV0JUI/pubchart?oid=644563241&format=interactive",
   sheets_gid="399976976",
   sql_file="well-known_security.sql"
   )
 }}
 
+{# TODO Anything to say about this graph? #}
+
```

Comment:
Anything to say here? Graph feels kind of just dumped there with no discussion on it.